FullStorySegmentMiddleware.podspec

```
@@ -1,38 +1,133 @@
 #
-# Be sure to run `pod lib lint FullStorySegmentMiddleware.podspec' to ensure this is a
-# valid spec before submitting.
+#  Be sure to run `pod spec lint FullStoryMiddleware.podspec' to ensure this is a
+#  valid spec and to remove all comments including this before submitting the spec.
 #
-# Any lines starting with a # are optional, but their use is encouraged
-# To learn more about a Podspec see https://guides.cocoapods.org/syntax/podspec.html
+#  To learn more about Podspec attributes see https://guides.cocoapods.org/syntax/podspec.html
+#  To see working Podspecs in the CocoaPods repo see https://github.com/CocoaPods/Specs/
 #
 
-Pod::Spec.new do |s|
-  s.name             = 'FullStorySegmentMiddleware'
-  s.version          = '0.1.0'
-  s.summary          = 'A short description of FullStorySegmentMiddleware.'
+Pod::Spec.new do |spec|
 
-# This description is used to generate tags and improve search results.
-#   * Think: What does it do? Why did you write it? What is the focus?
-#   * Try to keep it short, snappy and to the point.
-#   * Write the description between the DESC delimiters below.
-#   * Finally, don't worry about the indent, CocoaPods strips it!
+  # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  These will help people to find your library, and whilst it
+  #  can feel like a chore to fill in it's definitely to your advantage. The
+  #  summary should be tweet-length, and the description more in depth.
+  #
 
-  s.description      = <<-DESC
-TODO: Add long description of the pod here.
-                       DESC
+  spec.name         = "FullStorySegmentMiddleware"
+  spec.version      = "0.1.0"
+  spec.summary      = "FullStory Segment Middleware is a project demonstrating the full potential of joining forces between Segment + FullStory."
```

I think the summary should be a bit shorter.
```suggestion
  spec.summary      = "Demonstrates the full potential of joining forces between Segment and FullStory."
```